src/Components/FollowButton/MobileFollowArtistButton.tsx

```diff
@@ -0,0 +1,53 @@
+import React from "react"
+import { QueryRenderer, graphql } from "react-relay"
+import { SystemContextProps, withSystemContext } from "Artsy/SystemContext"
+import { Flex, Spinner } from "@artsy/palette"
+import {
+  MobileFollowArtistButtonQuery,
+  MobileFollowArtistButtonQueryResponse,
+  MobileFollowArtistButtonQueryVariables,
+} from "__generated__/MobileFollowArtistButtonQuery.graphql"
+import { FollowArtistButtonFragmentContainer as FollowArtistButton } from "./FollowArtistButton"
+import { ContextModule } from "@artsy/cohesion"
+
+export const MobileFollowArtistButton: React.FC<MobileFollowArtistButtonQueryResponse> = props => {
+  return (
+    <FollowArtistButton
+      contextModule={ContextModule.intextTooltip}
```

Comment:
Realizing this is not actually a tooltip -- @abhitip is there a context we should add for follow buttons within article text? 
<img width="703" alt="Screen Shot 2020-10-01 at 6 07 26 PM" src="https://user-images.githubusercontent.com/1497424/94868344-016fee80-0411-11eb-8951-cf83d3f0d020.png">

Reply:
Just added a new context module `articleArtist` and tagged @jonallured to merge! https://github.com/artsy/cohesion/pull/114

Reply:
Merged! So my next step would be to wait for this to roll out and then use that new value in this PR, correct?

Reply:
yup

Reply:
Done with fe24363.